packages/docusaurus-plugin-content-blog/src/authors.ts

```diff
@@ -148,10 +158,28 @@ ${Object.keys(authorsMap)
   return frontMatterAuthors.map(toAuthor);
 }
 
+function fixAuthorImageBaseURL(
+  authors: Author[],
+  {baseUrl}: {baseUrl: string},
+) {
+  authors.forEach((author) => {
```

Comment:
prefer FP style: use map and return new object instead of mutating input with side effect